content/quickstart/python/metrics.md

```diff
@@ -98,16 +95,10 @@ import opencensus.tags import tag_value as tag_value_module
 
 # Create the measures
 # The latency in milliseconds
-m_latency_ms = measure_module.MeasureFloat("repl/latency", "The latency in milliseconds per REPL loop", "ms")
-
-# Counts the number of lines read in from standard input
-m_lines_in = measure_module.MeasureInt("repl/lines_in", "The number of lines read in", "1")
-
-# Encounters the number of non EOF(end-of-file) errors.
-m_errors = measure_module.Int("repl/errors", "The number of errors encountered", "1")
+m_latency_ms = measure_module.MeasureFloat("repl_latency", "The latency in milliseconds per REPL loop", "ms")
 
 # Counts/groups the lengths of lines read in.
-m_line_lengths = measure_module.Int("repl/line_lengths", "The distribution of line lengths", "By")
+m_line_lengths = measure_module.Int("repl_line_lengths", "The distribution of line lengths", "By")
```

Comment:
This code won't run, is it supposed to be `measure_module.MeasureInt` instead of `measure_module.Int`

Reply:
I've fixed this when helping fix merge conflicts with code on the master branch.